apis/common/types.go

```diff
@@ -1,7 +1,26 @@
 package common
 
+import (
+	corev1 "k8s.io/api/core/v1"
+)
+
 type Image struct {
 	Repository string `json:"repository"`
 	Image      string `json:"image"`
 	Tag        string `json:"tag"`
 }
+
+type PrometheusConfig struct {
```

Comment:
Nice

Reply:
Doesn't work yet but I we will probably want to get these refactored into a common place.